changelog/unreleased/issue-2319

```diff
@@ -0,0 +1,12 @@
+Bugfix: Correctly dump directories into tar files
+
+The dump command previously wrote directories in a tar file in a way which
+can cause compatibility problems. This caused for example 7zip on Windows
+to open tar files containing directories. In addition it was not possible
+to dump directories with extended attributes. These compatibility problems
+were fixed by properly setting the attributes of a directory.
```

Comment:
Are now fixed: solution in present tense.

Reply:
Ok